Source/WebKit/WebProcess/GPU/graphics/RemoteImageBufferProxy.cpp

```diff
@@ -359,21 +361,15 @@ std::unique_ptr<SerializedImageBuffer> RemoteImageBufferProxy::sinkIntoSerialize
 {
     ASSERT(hasOneRef());
 
-    flushDrawingContext();
     m_remoteDisplayList.disconnect();
 
     RefPtr remoteRenderingBackendProxy = m_remoteRenderingBackendProxy.get();
     if (!remoteRenderingBackendProxy)
         return nullptr;
 
-    prepareForBackingStoreChange();
-
-    if (!ensureBackend())
-        return nullptr;
-
     remoteRenderingBackendProxy->remoteResourceCacheProxy().forgetImageBuffer(m_renderingResourceIdentifier);
 
-    auto result = makeUnique<RemoteSerializedImageBufferProxy>(parameters(), backendInfo(), m_renderingResourceIdentifier, *remoteRenderingBackendProxy);
+    auto result = makeUnique<RemoteSerializedImageBufferProxy>(parameters(), backendInfo(), m_renderingResourceReferenceTracker, m_needsFlush, *remoteRenderingBackendProxy);
```

Comment:
Adding a move constructor to the reference tracker, and using that rather than copying via `add` would make this easier to understand imo.

Reply:
I did this hack because the copy constructor was deleted (which makes sense to me).
Yeah, I think a move constructor is the right way to communicate what I'm doing here.
I'll fix this when I'm back at keyboard.